docs/community/README.md

```diff
@@ -0,0 +1,97 @@
+# How to Get Involved
+
+<!-- TODO: what is community, what are the touchpoints -->
+
+## Questions
+
+For the fastest response, you can ask questions on the [`#general`, `#serving`
+or `#eventing` channels of the Knative Slack](https://slack.knative.dev/).
+
+We also have a [Stack Overflow topic,
+`knative`](https://stackoverflow.com/questions/tagged/knative) (as well as
+`knative-serving` and `knative-eventing`), and the
```

Comment:
same as above

Reply:
I created the `knative-functions` tag on Stack Overflow just now, so now we can link to its presence